src/debug/debug-pal/unix/diagnosticsipc.cpp

```diff
@@ -105,10 +104,7 @@ IpcStream *IpcStream::DiagnosticsIpc::Accept(ErrorCallback callback) const
         return nullptr;
     }
 
-    auto pIpcStream = new (std::nothrow) IpcStream(clientSocket);
-    if (pIpcStream == nullptr && callback != nullptr)
-        callback("Failed to allocate an IpcStream object.", 1);
-    return pIpcStream;
+    return new IpcStream(clientSocket);
```

Comment:
Please don't use `new` / `delete` in PAL, you need to use `InternalNew<T>()` / `InternalDelete(...)` or malloc / free.

Reply:
Ah, I guess I got mislead by the PAL in the path. This is not the coreclr PAL, right? So please ignore my comment.